.tekton/push.yaml

```diff
@@ -21,6 +21,7 @@ spec:
     - name: infra-deployment-update-script
       value: |
         sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/.*?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/has/kustomization.yaml
+        sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/config/monitoring/?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/monitoring/grafana/base/kustomization.yaml
```

Comment:
Ah, wait I misread this.

Shouldn't this go to the HAS specific folder under https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring/grafana/base?

Reply:
sigh, looks like they refactored 3 days ago.  All components used to write into the kustomization file according to the instructions